compose.ci.sh

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env bash
+cd "$( dirname "$0" )"
+source ./.devcontainer/funcs.sh
+BPROIMG=rocky85-ci
```

Comment:
the .sh sets BPROIMG and that determines the public buildpro image that will be the base of the docker container